src/citra_qt/configure_graphics.ui

```diff
@@ -71,22 +81,5 @@
   </layout>
  </widget>
  <resources/>
- <connections>
```

Comment:
changed to <connections>, visual studio and appveyor get me error,so changed it back. the other ui files ends with <connection/>.